submission/persistence-common-api/src/main/kotlin/ac/uk/ebi/biostd/persistence/common/service/SubmissionOperations.kt

```diff
@@ -92,15 +96,13 @@ interface SubmissionRequestPersistenceService {
 
     fun getProcessingRequests(since: TemporalAmount? = null): Flow<Pair<String, Int>>
 
-    suspend fun updateRqtIndex(
-        accNo: String,
-        version: Int,
-        index: Int,
-    )
-
-    suspend fun updateRqtIndex(
-        requestFile: SubmissionRequestFile,
-        file: ExtFile,
+    /**
+     * Update the given request files. By default only file index is updated in submission request. For other options
+     * @see UpdateOptions
+     */
+    suspend fun updateRqtFile(
```

Comment:
this change will allow to change file status in future set of changes without chaning API